Projects/My_website/myWebsite/myWebsite/settings.py

```diff
@@ -134,3 +135,12 @@
 # https://docs.djangoproject.com/en/5.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': [
+        'rest_framework.authentication.BasicAuthentication',
+    ],
```

Comment:
@codal-uchoudhary , as discussed in person as well as our 1:1, you need to implement TokenAuthentication and not the Basic Authentication. 

So, pls update accordingly.

Reply:
Token based Authentication is Implemented in the Blog project